src/notation/internal/notationinteraction.cpp

```diff
@@ -2845,7 +2995,52 @@ void NotationInteraction::replaceSelectedNotesWithSlashes()
     notifyAboutNotationChanged();
 }
 
-void mu::notation::NotationInteraction::changeEnharmonicSpelling(bool both)
+void NotationInteraction::repeatSelection()
+{
+    const Ms::Selection& selection = score()->selection();
```

Comment:
copied from ScoreView::cmdRepeatSelection, with some simplifications (actually I'm pretty sure it can be simplified even further)